tests/test_error.py

```diff
@@ -52,6 +52,14 @@ def test_repr(self):
                 "request_id='123')"
             )
 
+    def test_error_object(self):
+        err = error.StripeError(
+            "message", json_body={"error": {"code": "some_error"}}
+        )
```

Comment:
@remi-stripe This PR was a breaking change for us since we had code similar to this in our own test suite. The difference was our `json_body.error` was a string, not an object:

```
stripe.error.StripeError('hi', json_body={"error": "Test json_body error"})
```

Starting on 2.37.0 this fails with the following:

```
    json_body={"error": "Test json_body error"},
/usr/local/lib/python3.7/site-packages/stripe/error.py:140: in __init__
    message, http_body, http_status, json_body, headers, code
/usr/local/lib/python3.7/site-packages/stripe/error.py:36: in __init__
    self.error = self.construct_error_object()
/usr/local/lib/python3.7/site-packages/stripe/error.py:66: in construct_error_object
    self.json_body["error"], stripe.api_key
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    @classmethod
    def construct_from(
        cls,
        values,
        key,
        stripe_version=None,
        stripe_account=None,
        last_response=None,
    ):
        instance = cls(
>           values.get("id"),
            api_key=key,
            stripe_version=stripe_version,
            stripe_account=stripe_account,
            last_response=last_response,
        )
E       AttributeError: 'str' object has no attribute 'get'
```

Just an FYI, not sure whether you'd like to fix this or publish a notice in the CHANGELOG or something. But the gist is that we updated a minor version and it was a breaking change for us.

Reply:
Sorry, pinged the wrong person @ob-stripe

Reply:
@thieman Thanks for the feedback. I think we can fix with a slightly stricter check (i.e. check that `json_body['error']` is a `dict` before trying to use it to construct the `StripeError`).

Reply:
Should be fixed in 2.37.1.